hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -284,6 +290,13 @@ public class HoodieTableConfig extends HoodieConfig {
       .sinceVersion("0.13.0")
       .withDocumentation("The metadata of secondary indexes");
 
+  public static final ConfigProperty<String> TABLE_BASE_FILE_FORMATS = ConfigProperty
+      .key("hoodie.table.base.file.formats")
+      .defaultValue(HoodieFileFormat.PARQUET.name())
+      .markAdvanced()
+      .sinceVersion("1.0.0")
+      .withDocumentation("Comma-separated list of base file formats in this Hudi table.");
```

Comment:
I'm confused a lot by this option, do you mean allowable file formats in total or currently file formats already in use?

Reply:
my bad.. i removed all the usages of this config but missed removing the config itself! will remove it.